src/Makefile.in

```diff
@@ -56,8 +77,9 @@ ifeq ($(VERILATOR_NO_OPT_BUILD),1)  # Faster laptop development... One build
 	-mv -f $@.tmp $@
 else
 ../bin/verilator_bin$(EXEEXT): obj_opt ../bin prefiles
-	$(MAKE) -C obj_opt -j 1  TGT=../$@ -f ../Makefile_obj serial
-	$(MAKE) -C obj_opt       TGT=../$@ -f ../Makefile_obj
+	$(RM_OBJ_OPT_COMPILE_COMMANDS)
+	$(BEAR_OBJ_OPT) $(MAKE) -C obj_opt -j 1  TGT=../$@ -f ../Makefile_obj serial
```

Comment:
> But if the submakes do nothing, does this break? https://github.com/verilator/verilator/pull/4228#discussion_r1317969099

I'm not sure what you meant by this. Could you please elaborate?

If you meant that there will be wrong `compile_commands.json` file when bear will ran on already built target, I've fixed this case. We didn't need to remove `compile_commands.json` file before generating it again, as we pass ``--append`` flag to ``bear``.